routers/api/actions/runner/utils.go

```diff
@@ -68,6 +68,11 @@ func getSecretsOfTask(ctx context.Context, task *actions_model.ActionTask) map[s
 		return secrets
 	}
 
+	globalSecrets, err := db.Find[secret_model.Secret](ctx, secret_model.FindSecretsOptions{})
```

Comment:
Maybe we should have a `type` column of `FindSecretsOptions`, otherwise it will make confusion that this is fetching all the secrets or only global secrets.

Reply:
Updated :)